lib/Sema/DerivedConformanceComparable.cpp

```diff
@@ -0,0 +1,411 @@
+//===--- DerivedConformanceComparable.cpp - Derived Comparable -===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
```

Comment:
This should be `// Copyright (c) 2019 - 2020` I think; and the other files should be updated too.